tests/system/gapic/v1beta1/test_system_tables_client_v1.py

```diff
@@ -87,6 +90,11 @@ def test_create_delete_dataset(self):
         client.delete_dataset(dataset=dataset)
 
     @vpcsc_config.skip_if_inside_vpcsc
+    @backoff.on_exception(
+        wait_gen=lambda: iter([MAX_WAIT_TIME_SECONDS, 120, BACKOFF_TIMEOUT]),
```

Comment:
I think the variable names are a little confusing, and 120 isn't covered by a constant.

Another option to consider is to use a list? For example:
```python
MAX_WAIT_TIME_SECONDS=[60, 120, 240]
...
wait_gen=lambda: iter(MAX_WAIT_TIME_SECONDS),
...
max_tries=len(MAX_WAIT_TIME_SECONDS)
```